### PR TITLE
AAE-21097 Implement permission-based authorization

### PR DESCRIPTION
**Pull request by tom-dal**
Issue link: AAE-22123

### CODE REVIEW

File: activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java [hunk 1]
```diff
@@ -34,8 +34,11 @@ public JwtGrantedAuthorityConverter(JwtAccessTokenProvider jwtAccessTokenProvide
     @Override
     public Collection<GrantedAuthority> convert(Jwt jwt) {
         Collection<GrantedAuthority> grantedAuthorities = new ArrayList<>();
-        for (String authority : jwtAccessTokenProvider.accessToken(jwt).getRoles()) {
-            grantedAuthorities.add(new SimpleGrantedAuthority(this.authorityPrefix + authority));
+        for (String role : jwtAccessTokenProvider.accessToken(jwt).getRoles()) {
+            grantedAuthorities.add(new SimpleGrantedAuthority(this.authorityPrefix + role));
+        }
+        for (String permission : jwtAccessTokenProvider.accessToken(jwt).getPermissions()) {
```
Comment (jesty) [inline activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java:40]: should u use PERMISSION_?

**Review by jesty — CHANGES_REQUESTED** [review]

File: activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java (hunk 1 above)
Comment (tom-dal) [reply activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java:40]: We can use PERMISSION_ but then we should override default behaviour of Spring which uses ROLE_ for both roles and authorities (there's no concept of permission in the AuthorizationManager).

Comment (tom-dal) [reply activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java:40]: We can use PERMISSION_ but then we should override default behaviour of Spring which uses ROLE_ for both roles and authorities (there's no concept of permission in the AuthorizationManager).

**Review by jesty — APPROVED** [review]
LGTM

**Pull request merged**